app/models/miq_worker.rb

```diff
@@ -90,9 +90,10 @@ def self.concrete_subclasses
     leaf_subclasses | descendants.select { |d| d.try(:acts_as_sti_leaf_class?) }
   end
 
-  class_attribute :default_queue_name, :required_roles, :maximum_workers_count, :include_stopping_workers_on_synchronize
+  class_attribute :default_queue_name, :required_roles, :maximum_workers_count, :include_stopping_workers_on_synchronize, :worker_settings_paths
   self.include_stopping_workers_on_synchronize = false
   self.required_roles = []
+  self.worker_settings_paths = []
```

Comment:
Not sure about this name, we have `config_settings_path` already which is the full path to the worker settings: `[:workers] + path_to_my_worker_settings` but this doesn't seem to be used.